bucket/prismlauncher-qt5.json

```diff
@@ -3,12 +3,27 @@
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-6.3.zip",
-    "hash": "003b54560935c341ab825d41962323e56d649ea15a29f69d4ce64b93326c866c",
+    "notes": [
+        "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically. If you are using a global install, you will need to copy your own data to the new Scoop persist directory"
```

Comment:
I think this would be better if it was reworded a bit:
```json
        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically."
        "If this package can't determine which data to migrate, you will need to manually copy your data from AppData to the new Scoop PrismLauncher persist directory."
```

Reply:
i adopted some of these suggestions as well :)